features/old/api/features.feature

```diff
@@ -8,59 +8,60 @@ Feature: Features :)
     And an application plan "Basic" of provider "foo.3scale.localhost"
     Given current domain is the admin domain of provider "foo.3scale.localhost"
 
-  @javascript @wip
+  @javascript
   Scenario: Create a feature
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
     When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
+    Then I go to the edit page for plan "Basic"
     And I follow "New feature"
     And I fill in "Name" with "Free T-shirt"
+    And I fill in "System name" with "tee"
     And I fill in "Description" with "T-shirt with logo of our company for free."
-    And I press "Create Feature"
-    Then I should see enabled feature "Free T-shirt"
-    And provider "foo.3scale.localhost" should have feature "Free T-shirt"
-    And feature "Free T-shirt" should be enabled for plan "Basic"
+    And I press "Save"
+    Then I should see "T-shirt with logo of our company for free."
 
   # TODO: Test that the "No features yet" notice dissapears when first feature is created,
   #       and appears when last one is deleted.
-
-  @javascript @wip
+  @javascript
   Scenario: Disable a feature
-    Given a feature "50% less bugs" of provider "foo.3scale.localhost"
-    And feature "50% less bugs" is enabled for plan "Basic"
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
+    And a feature "50% more bugs" of provider "foo.3scale.localhost"
+
     When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
-    And I press the disable button for feature "50% less bugs"
-    Then I should see disabled feature "50% less bugs"
-    And feature "50% less bugs" should be disabled for plan "Basic"
+    Then I go to the edit page for plan "Basic"
+    And I disable feature "50% more bugs"
+    Then feature "50% more bugs" should be disabled
 
-  @javascript @wip
+  @javascript
   Scenario: Enable a feature
-    Given a feature "Light-speed connection" of provider "foo.3scale.localhost"
-    And feature "Light-speed connection" is disabled for plan "Basic"
+    Given a service plan "Basic" of provider "foo.3scale.localhost"
+    And a feature "50% less bugs" of provider "foo.3scale.localhost"
+
     When I log in as provider "foo.3scale.localhost"
-    And I go to the edit page for plan "Basic"
-    And I press the enable button for feature "Light-speed connection"
-    Then I should see enabled feature "Light-speed connection"
-    And feature "Light-speed connection" should be enabled for plan "Basic"
+    Then I go to the edit page for plan "Basic"
```

Comment:
This goes to the Application plan Basic page, but then the other steps are for service plans... this is not right.

Reply:
correct

Reply:
Actually this is changed if you see in step and I forgot to update here but good you pointed it